packages/core/src/App/Containers/VerificationModal/verification-modal.scss

```diff
@@ -0,0 +1,513 @@
+
+                        @include mobile {
+                            margin: 1.6rem 0;
+                            padding: 1.6rem;
+                            width: 100%;
+                        }
+                    }
+                }
+
+                &-icon {
+                    width: 12.8rem;
+                    height: 12.8rem;
+                    margin-top: 8rem;
+
+                    @include mobile {
+                        margin-top: 4rem;
+
+                        /* iPhone SE screen height fixes due to UI space restrictions */
+                        @media only screen and (max-height: 480px) {
+                            margin-top: 3.2rem;
+                        }
+                    }
+                }
+
+                &-title {
+                    font-size: var(--text-size-xs);
+                    margin-bottom: 1rem;
+                    text-align: center;
+                }
+
+                &-subtitle {
+                    font-size: var(--text-size-xxs);
+                    font-weight: 400;
+                    text-align: center;
+                    line-height: 1.5;
+                }
+
+                &-wrapper {
+                    display: flex;
+                    justify-content: center;
+                    width: 100%;
+
+                    &-full-width {
+                        width: calc(100vw - 33rem);
+
+                        @include mobile {
+                            width: 100%;
+                        }
+                    }
+
+                    @include mobile {
+                        overflow-x: hidden;
+                        overflow-y: auto;
+                    }
+                }
+
+                @include mobile {
+                    line-height: 2rem;
+                    text-align: center;
+                }
+            }
+        }
+    }
+
+    .min-height {
+        min-height: 50vh;
+
+        @include mobile {
+            min-height: unset;
+        }
+    }
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: center;
+
+        @include mobile {
+            min-height: unset;
+            overflow-y: scroll;
+            justify-content: center;
+        }
+
+        &--reset {
+            align-items: normal;
+            @include mobile {
+                overflow-y: unset;
+                padding: 1.6rem;
+            }
+            .proof-of-identity__submit-button {
+                @include mobile {
+                    margin: unset;
+                    width: 100%;
+                }
+            }
+            .account-form__footer {
+                @include mobile {
+                    padding: 1.6rem 2.4rem;
+                }
+            }
+            .form-body {
+                @include desktop {
+                    z-index: 5;
+                }
+            }
+        }
+
+        .icon {
+            width: 12.8rem;
```

Comment:
@yauheni-deriv  we have to declare this css here because we are passing in the container of POI and the default size of the icon doesn't match the size of the design. It's better to change the css here instead of changing the main component icon size. 🙏